resources/js/lang/de/rooms.js

```diff
@@ -36,6 +36,14 @@ export default {
       title: 'Zugriff verweigert'
     }
   },
+  modals: {
+    delete: {
+      title: 'Raum löschen',
+      confirm: 'Soll der Raum "{name}" gelöscht werden?',
+      yes: 'Ja',
```

Comment:
This translation and the following one should be moved to a central place, to make it dry...